src/program/lwaftr/query/selftest.sh

```diff
@@ -51,35 +58,29 @@ function fatal {
 }
 
 function test_lwaftr_query {
-    local pid=$1
-    # FIXME: Sometimes lwaftr query gets stalled. Add timeout.
-    local lineno=`timeout 1 ./snabb lwaftr query $pid | wc -l`
+    ./snabb lwaftr query $@ > $TEMP_FILE
+    local lineno=`cat $TEMP_FILE | wc -l`
     if [[ $lineno -gt 1 ]]; then
-        echo "Success: lwaftr query $pid"
+        echo "Success: lwaftr query $@"
     else
-        fatal "lwaftr query $pid"
-    fi
-}
-
-function test_lwaftr_query_filter {
-    local pid=$1
-    local filter=$2
-    local lineno=`timeout 1 ./snabb lwaftr query $pid $filter | wc -l`
-    if [[ $lineno -gt 1 ]]; then
-        echo "Success: lwaftr query $pid $filter"
-    else
-        fatal "lwaftr query $pid"
+        fatal "lwaftr query $@"
     fi
 }
 
 # Run lwAFTR.
-tmux_launch "lwaftr" "./snabb lwaftr run --reconfigurable --conf $LWAFTR_CONF --v4 $SNABB_PCI0 --v6 $SNABB_PCI1"
+tmux_launch "lwaftr" "./snabb lwaftr run --reconfigurable --name lwaftr --conf $LWAFTR_CONF --v4 $SNABB_PCI0 --v6 $SNABB_PCI1" "lwaftr.log"
```

Comment:
I believe @wingo has previously mentioned how hardcoded names like this will cause race conditions when we're running test suites in parallel under hydra. We need a random name.